src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(int length, int width) {
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, `createRandomFigure` should not be in the `Figure` class hierarchy if it's not a common behavior for all figures. Consider refactoring this to a more appropriate place, such as `FigureSupplier`.